omnibus/files/private-chef-cookbooks/private-chef/spec/libraries/private_chef_spec.rb

```diff
@@ -165,7 +165,8 @@ def config_for(hostname)
     end
   end
 
-  context "in an HA topology" do
+# HA is deprecated
+  context "in an HA topology", focus: true do
```

Comment:
`focus: true` needs to be removed before merge

Reply:
Crud. Thanks for spotting that.